pontoon/sync/templatetags/helpers.py

```diff
@@ -0,0 +1,17 @@
+from django_jinja import library
+
+
+@library.filter
+def sync_log_projects(sync_log):
+    projects = [log.project for log in sync_log.project_sync_logs.all()]
+    if len(projects) == 0:
+        return 'Unknown'
+    elif len(projects) == 1:
+        return projects[0].name
+    elif len(projects) < 4:
+        return ', '.join(p.name for p in projects)
+    else:
+        return '{names} and {total} more'.format(
+            names=', '.join(p.name for p in projects[:3]),
+            total=len(projects) - 3
+        )
```

Comment:
We'll be syncing all projects most of the time, so "All projects" should be added here.